tests/liquidation_test.py

```diff
@@ -1,4 +1,74 @@
 import unittest
+from web3 import Web3
+
+from mev_inspect.aave_liquidations import get_liquidations
+from mev_inspect.schemas.liquidations import Liquidation
+from mev_inspect.schemas.classified_traces import Protocol
+from mev_inspect.classifiers.trace import TraceClassifier
+from mev_inspect.block import create_from_block_number
+
+
+class TestAaveLiquidations(unittest.TestCase):
+    def test_single_weth_liquidation(self):
+
+        base_provider = Web3.HTTPProvider("http://162.55.96.141:8546/")
```

Comment:
we shouldn't be doing any data fetching in tests in case they get run offline / away from where the service can be contacted

check out the arbitrage integration tests for how we leverage saved blocks

Reply:
Please be careful to not hardcode nodes like this. If possible scrub that node out of the commit history entirely.

Reply:
removed